script/fetch-issues.js

```diff
@@ -141,23 +192,132 @@ async function fetchNextPage(owner, name, isIssue, data, cursor, pageCount) {
         }
       };
       if (!isIssue) {
-        info.pullRequest = {
-          htmlUrl: node.url
+        let rawComments, comments;
+        // collect review comments
+        rawComments = node.reviews.nodes.map(node => node.comments.nodes);
+        rawComments = [].concat.apply([], rawComments);
+        // collect issue comments
+        rawComments = rawComments.concat(node.comments.nodes);
+
+        // fetch reactions only if there are reactions within that PR
+        // to reduce API hits.
+        // trick: a comment has received reaction(s) iff
```

Comment:
`iff` -> `if` ?

Reply:
Nope, that was intentional. `iff` = `if and only if`